include/deal.II/base/array_view.h

```diff
@@ -193,6 +193,11 @@ class ArrayView
    */
   std::size_t size() const;
 
+  /** Returns a pointer to the underlying array serving as element storage.
```

Comment:
Put the text on a separate line. I think we usually also use the form "Return a pointer" instead of "Returns a pointer". (I know there are many places in the library where we use the latter, but it's probably still 80:20.)